packages/components/src/components/navigation/Link/Link.tsx

```diff
@@ -1,24 +1,24 @@
-import React from 'react';
-import { forwardRef, IconProps, Link as ChakraLink, ResponsiveValue } from '@chakra-ui/react';
+import React, { ReactElement, cloneElement, isValidElement } from 'react';
+import {
+  forwardRef,
+  IconProps,
+  Link as ChakraLink,
+  ResponsiveValue,
+  useStyleConfig,
+  StyleProps,
+  chakra,
+} from '@chakra-ui/react';
 import { LinkProps, LinkSize } from './types';
 import { getIconSize } from './getIconSize';
 
 export const Link = forwardRef<LinkProps, typeof ChakraLink>(
-  ({ children, leadingIcon, trailingIcon, size = 'm', variant, ...props }, ref) => {
-    const leftIcon = getIcon(leadingIcon, size);
-    const rightIcon = getIcon(trailingIcon, size);
+  ({ children, leadingIcon, trailingIcon, size = 'm', variant, isInline = false, ...props }, ref) => {
+    const styles = useStyleConfig('Link', { isInline, variant, size }) as StyleProps;
+    const leftIcon = getIcon(leadingIcon, size, isInline);
+    const rightIcon = getIcon(trailingIcon, size, isInline);
 
     return (
-      <ChakraLink
-        ref={ref}
-        size={size}
-        variant={variant}
-        data-testid="cmpsr.link.container"
-        alignItems="center"
-        columnGap="0.5rem"
-        width="inherit"
-        {...props}
-      >
+      <ChakraLink ref={ref} data-testid="cmpsr.link.container" {...styles} {...props}>
```

Comment:
I know it was already in here, but do we really need testid or can we use something "more semantic" to reference the elements?